Allos.Amazon.Sdk/Runtime/StreamBytesReadEventArgs.cs

```diff
@@ -8,8 +8,11 @@ namespace Allos.Amazon.Sdk;
 public partial class EventStream : WrapperStream
 {
     /// <summary>
-    /// Adapted from internal EventStream.ReadProgress delegate
+    /// Event arguments for <see cref="OnRead"/>
```

Comment:
This class has blank Dispose(). Suggestion:

```
protected override void Dispose(bool disposing)
{
    if (disposing)
    {
         OnRead = null;
    }

     base.Dispose(disposing);
}
```

Reply:
good call;  this actually seems like it might be a bug in the actual SDK from which this was taken.  In any case, `base.Dispose(bool)` is not ideal either, but in a more recent commit I have changed this to:

```csharp
protected override void Dispose(bool disposing)
{
    if (disposing)
    {
        OnRead = null;
    }

    //`base.Dispose(disposing)` will always do `BaseStream.Close()` which is not what we want,
    //instead we will call `Close()` directly
    Close();
}
```